### PR TITLE
feat(switch): add loading attribute

### PR DESCRIPTION
**Pull request by MBearo**
- [ ] Component Migration

Fix #

### CODE REVIEW

File: packages/switch/src/index.vue [hunk 1]
```diff
@@ -195,6 +203,8 @@ export default defineComponent({
       const coreEl = core.value
       coreEl.style.borderColor = newColor
       coreEl.style.backgroundColor = newColor
+      console.log(coreEl.children[0])
```
Comment (JeremyWuuuuu) [inline packages/switch/src/index.vue:206]: You need to remove this line.

**Review by jw-foss — COMMENTED** [review]
I mean, what is the point of adding an attribute like this? We have provided `v-loading` directive that you can easily apply upon any components that you have to give the component `loading` effect. Why adding this anyway?

**Comment by JeremyWuuuuu** [comment]
What kind of problem does this solve? https://ant.design/components/switch/ according to `AntDesign` they had a specific design on this one, but for now we do not have resources for UI Design, I personally cannot give the best suggestion here in order to match the original design language, I do not know if this change breaks the consistency of `ElementUI`'s design language.

**Comment by MBearo** [comment]
> What kind of problem does this solve? https://ant.design/components/switch/ according to `AntDesign` they had a specific design on this one, but for now we do not have resources for UI Design, I personally cannot give the best suggestion here in order to match the original design language, I do not know if this change breaks the consistency of `ElementUI`'s design language.

Ok, maybe I don't fully understand `Element UI`'s design language. My idea is to treat the `switch` component as a special `button` component, switch between true and false, and also have a loading attribute. Just to give users a more delicate experience.

**Comment by zouhangwithsweet** [comment]
I think it's a good feature. Can you show us a gif of the `loading switch`?

**Comment by zouhangwithsweet** [comment]
I think it's a good feature. Can you show us a gif of the `loading switch`?

**Comment by MBearo** [comment]
Screen Recording 2020-10-19 at 09

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
Alright, merged. Nice feature!

**Pull request merged**